doc/mgr/dashboard.rst

```diff
@@ -366,6 +366,63 @@ To enable SSO::
 
   $ ceph dashboard sso enable saml2
 
+Enabling Prometheus alerting
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
Please add a short note about what the Prometheus Alert manager is all about, including an URL that points to more information, especially how to install and configure it.